packages/sandpack-core/src/npm/merge-dependency.ts

```diff
@@ -163,20 +165,27 @@ export function mergeDependencies(responses: ILambdaResponse[]) {
         d => d.name === depDepName
       );
 
-      if (
-        rootDependency &&
-        !intersects(rootDependency.version, newDepDep.semver) &&
-        // If the resolved or semver is the same we don't have to actually replace dependency info
-        rootDependency.version !== newDepDep.resolved &&
-        rootDependency.version !== newDepDep.semver &&
-        rootDependency.name !== r.dependency.name // and this dependency doesn't require an older version of itself
-      ) {
-        // If a root dependency is in conflict with a child dependency, we always
-        // go for the root dependency
-        replaceDependencyInfo(r, depDepName, newDepDep);
-
-        // Start from the beginning, to make sure everything is correct
-        i = -1;
+      if (rootDependency) {
+        // packages that require themselves?
+        if (r.dependency.name !== depDepName) {
```

Comment:
Ah, yes this is interesting. There are packages that do an import by literally importing from themselves. So that's a case we need to explicitly support 😂

Reply:
For example (let's say package is `@babel/runtime`):

```js
import { isImport } from '@babel/runtime/utils/import';
```

Reply:
Oh wow, didn't know it actually was a thing, but good that we handle it than :D